README.md

```diff
@@ -1,9 +1,42 @@
 
 # authd
 
-## Database setup
+## Requirement installation
+
+- crystal
+- shards
+- postgresql
+
+Usually install on Aarch like
+```shell
+sudo pacman -S crystal shards git vim postgresql
+```
+## Database init
+```shell
+sudo su -l postgres
+initdb -D '/var/lib/postgres/data'
+sudo systemctl start postgresql.service
+createuser --interactive 
+#create <dbuser>
+createdb <database>
+```
 
+## Database setup
+```shell
+psql -d <database>
+```
 ```sql
 create table users(id int, created_at date, updated_at date, username text, realname text, password text, avatar text, perms text[]);
 ```
 
+## start
+
+```shell
+git clone <branch>
```

Comment:
Should actually be the URL in this command, not the branch.

Those are instructions to clone a git repository and build its content. I’m not sure the cloning part is necessary to include, but I’m not against it. Most of the following instructions are about building it though, so maybe rename the section from “start” to “Build & Run” or something similar?